### PR TITLE
[releng] Add release 2.2 to Release Compatibility Matrix for PyTorch releases

### PR DESCRIPTION
**Pull request by atalman**
Update RELEASE.md for release 2.2

### CODE REVIEW

**Review by DanilBaibak — APPROVED** [review]

**Comment by atalman** [comment]
> > Update RELEASE.md for release 2.2
> 
> Not 12.2 support?

@johnnynunez We are keeping same CUDa matrix for 2.2 release. Most likely CUDA version will be updated for Release 2.3.

**Comment by johnnynunez** [comment]
> > > Update RELEASE.md for release 2.2
> > 
> > 
> > Not 12.2 support?
> 
> @johnnynunez We are keeping same CUDa matrix for 2.2 release. Most likely CUDA version will be updated for Release 2.3.

thanks. Update to cuda 12.3 or future 12.4 directly

File: RELEASE.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Following is the Release Compatibility Matrix for PyTorch releases:
 
 | PyTorch version | Python | Stable CUDA | Experimental CUDA |
 | --- | --- | --- | --- |
+| 2.2 | >=3.8, <=3.11 | CUDA 11.8, CUDNN 8.7.0.84 | CUDA 12.1, CUDNN 8.9.2.26 |
```
[the remark this answers is not in the document]
Comment (atalman) [reply RELEASE.md:50]: Yes its available in nightly wheel only now. No coda and  No domain support: https://hud.pytorch.org/hud/pytorch/pytorch/nightly/1?per_page=50&name_filter=3_12

**Pull request closed without merging**